ext/native/thin3d/thin3d.cpp

```diff
@@ -352,6 +353,35 @@ DrawContext::~DrawContext() {
 	DestroyPresets();
 }
 
+void DrawContext::RotateRectToDisplay(FRect &rect, float curRTWidth, float curRTHeight) {
+	if (g_display_rotation == DisplayRotation::ROTATE_0)
+		return;
+	switch (g_display_rotation) {
+	case DisplayRotation::ROTATE_180:
+		rect.x = curRTWidth - rect.w - rect.x;
+		rect.y = curRTHeight - rect.h - rect.y;
+		break;
+	case DisplayRotation::ROTATE_90: {
+		// Note that curRTWidth_ and curRTHeight_ are "swapped"!
+		float origX = rect.x;
+		float origY = rect.y;
+		float rtw = curRTHeight;
+		float rth = curRTWidth;
+		rect.x = rth - rect.h - origY;
+		rect.y = origX;
+		std::swap(rect.w, rect.h);
+		break;
+	}
+	case DisplayRotation::ROTATE_270: {
+		float origX = rect.x;
+		float origY = rect.y;
+		// TODO
+		std::swap(rect.w, rect.h);
```

Comment:
Hm, I copy-pasted this from the D3D11 code and didn't notice this case wasn't fully implemented.

Will do that before merge.